test/test-install.js

```diff
@@ -0,0 +1,37 @@
+'use strict'
+
+var test = require('tape')
+var install = require('../lib/install').test.install
+
+test('EACCES retry once', function (t) {
+  t.plan(3)
+
+  var fs = {}
+  fs.stat = function (path, cb) {
+    var err = new Error()
+    err.code = 'EACCES'
+    cb(err)
+    t.ok(true);
+  }
+
+
+  var gyp = {}
+  gyp.devDir = __dirname
+  gyp.opts = {}
+  gyp.opts.ensure = true
+  gyp.commands = {}
+  gyp.commands.install = function (argv, cb) {
+    install(fs, gyp, argv, cb)
+  }
+  gyp.commands.remove = function (argv, cb) {
+    cb()
+  }
+
+  gyp.commands.install([], function (err) {
+    t.ok(true)
```

Comment:
Could this test be removed and the test count reduced to 2?

Reply:
Well, you want to know that the final callback is called, right?  Or do I misunderstand you?

Reply:
Fair point.